### PR TITLE
[Edit] JavaScript Add Codebytes Issue: #2405

### LINKED ISSUE #2405: [Edit] JavaScript Add Codebytes
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

Please add Codebytes to the listed entries.
https://www.codecademy.com/resources/docs/javascript/map/delete
https://www.codecademy.com/resources/docs/javascript/map/get

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) when working on the PR for this issue.

Bear in mind the following two principles:
1) There should only be one Codebyte per entry listed as `## Codebyte Example` at the end of the entry.
2) The codebytes should be new examples, not copies of the prior examples.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by dghalbr**
### Description

Closes #2405 
Added Codebyte Examples to the following entries.
https://www.codecademy.com/resources/docs/javascript/map/delete
https://www.codecademy.com/resources/docs/javascript/map/get

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/map/terms/delete/delete.md [hunk 1]
```diff
@@ -37,3 +37,33 @@ console.log(fruits.delete('Strawberries')); // Output: false
 ```
 
 The first `.delete()` statement returns `true` because `fruits` contain an entry with `Oranges` as a key and has successfully removed it. The second statement returns `false` because an entry with the `Strawberries` key does not exist in `fruits`.
+
+## Codebyte Example
+
+In the example below, we declare a new inventory `Map` of consumable construction supplies linking them to their current quantities on hand. Throughout this program:
```
Comment (SSwiniarski) [inline content/javascript/concepts/map/terms/delete/delete.md:43]:
```suggestion
In the example below, we declare a new inventory `Map` of consumable construction supplies linking them to their current quantities on hand. Throughout this example:
```

File: content/javascript/concepts/map/terms/get/get.md [hunk 2]
```diff
@@ -36,3 +36,29 @@ console.log(fruits.get('Apples')); // Output: 5
 console.log(fruits.get('Oranges')); // Output: 8
 console.log(fruits.get('Bananas')); // Output: undefined
 ```
+
+## Codebyte Example
+
+In the example below, we declare a new `Map` of purchase orders.
+Throughout this program:
```
Comment (SSwiniarski) [inline content/javascript/concepts/map/terms/get/get.md:43]:
```suggestion
Throughout this example:
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@dghalbr I did a first review and had a couple of comments.

File: content/javascript/concepts/map/terms/delete/delete.md (hunk 1 above)
Comment (dghalbr) [reply content/javascript/concepts/map/terms/delete/delete.md]: Good catch. Updated.

File: content/javascript/concepts/map/terms/get/get.md (hunk 2 above)
Comment (dghalbr) [reply content/javascript/concepts/map/terms/get/get.md]: Again, good catch! Updated.

**Review by SSwiniarski — APPROVED** [review]
@dghalbr Looks good for a second review.

File: content/javascript/concepts/map/terms/delete/delete.md [hunk 3]
```diff
@@ -37,3 +37,33 @@ console.log(fruits.delete('Strawberries')); // Output: false
 ```
 
 The first `.delete()` statement returns `true` because `fruits` contain an entry with `Oranges` as a key and has successfully removed it. The second statement returns `false` because an entry with the `Strawberries` key does not exist in `fruits`.
+
+## Codebyte Example
+
+In the example below, we declare a new inventory `Map` of consumable construction supplies linking them to their current quantities on hand. Throughout this example:
```
Comment (caupolicandiaz) [inline content/javascript/concepts/map/terms/delete/delete.md:43]:
```suggestion
In the code below, a new inventory `Map` of consumable construction supplies catalogs the current quantities on hand. In this example:
```

Comment (caupolicandiaz) [reply content/javascript/concepts/map/terms/delete/delete.md:43]: generally we avoid the use of first and second person pronouns in docs (I, you, we)

File: content/javascript/concepts/map/terms/delete/delete.md [hunk 4]
```diff
@@ -37,3 +37,33 @@ console.log(fruits.delete('Strawberries')); // Output: false
...
+
+- Bob will log that he used a roll of tape by updating its value using the [`.set()` method](https://www.codecademy.com/resources/docs/javascript/map/set).
+- Bob noticed he used the last roll so he will remove it from the map using the [`.delete()` method](https://www.codecademy.com/resources/docs/javascript/map/delete). He feels confident it worked correctly due to `true` being displayed in the console output.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/map/terms/delete/delete.md:46]:
```suggestion
- Bob noticed he used the last roll so he will remove it from the map using the `.delete()` method. He feels confident it worked correctly due to `true` being displayed in the console output.
```

File: content/javascript/concepts/map/terms/delete/delete.md [hunk 5]
```diff
@@ -37,3 +37,33 @@ console.log(fruits.delete('Strawberries')); // Output: false
...
+- Manny notices that there are no Tape Rolls left, but he doesn't know Bob already called the [`.delete()` method](https://www.codecademy.com/resources/docs/javascript/map/delete). He attemps to delete it and notices the console outputs `false`.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/map/terms/delete/delete.md:47]:
```suggestion
- Manny notices that there are no Tape Rolls left, but he doesn't know Bob already called the `.delete()` method. He attemps to delete it and notices the console outputs `false`.
```

File: content/javascript/concepts/map/terms/delete/delete.md [hunk 6]
```diff
@@ -37,3 +37,33 @@ console.log(fruits.delete('Strawberries')); // Output: false
...
+]);
+
+console.log('Inventory map size: ', inventory.size);
+
+console.log('Bob uses a tape roll.');
+inventory.set('Tape Roll', '0');
+console.log('We ran out of tape ', inventory.get('Tape Roll'));
+
+console.log('Deleting tape from the map...');
+console.log(inventory.delete('Tape Roll')); // Output: true
```
Comment (caupolicandiaz) [inline content/javascript/concepts/map/terms/delete/delete.md:63]:
```suggestion
console.log(inventory.delete('Tape Roll')); 
```

Comment (caupolicandiaz) [reply content/javascript/concepts/map/terms/delete/delete.md:63]: this is a runnable code block, so, we don't need to comment the output

File: content/javascript/concepts/map/terms/delete/delete.md [hunk 7]
```diff
@@ -37,3 +37,33 @@ console.log(fruits.delete('Strawberries')); // Output: false
...
+console.log('Updated inventory map size: ', inventory.size);
+
+console.log('Deleting tape again from the map...');
+console.log(inventory.delete('Tape Roll')); // Output: false
```
Comment (caupolicandiaz) [inline content/javascript/concepts/map/terms/delete/delete.md:67]:
```suggestion
console.log(inventory.delete('Tape Roll'));
```

File: content/javascript/concepts/map/terms/get/get.md [hunk 8]
```diff
@@ -36,3 +36,29 @@ console.log(fruits.get('Apples')); // Output: 5
 console.log(fruits.get('Oranges')); // Output: 8
 console.log(fruits.get('Bananas')); // Output: undefined
 ```
+
+## Codebyte Example
+
+In the example below, we declare a new `Map` of purchase orders.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/map/terms/get/get.md:42]:
```suggestion
The code below uses a new `Map` that contains purchase orders.
```

File: content/javascript/concepts/map/terms/get/get.md [hunk 9]
```diff
@@ -36,3 +36,29 @@ console.log(fruits.get('Apples')); // Output: 5
...
+Throughout this example:
```
Comment (caupolicandiaz) [inline content/javascript/concepts/map/terms/get/get.md:43]:
```suggestion
In this example:
```

File: content/javascript/concepts/map/terms/get/get.md [hunk 10]
```diff
@@ -36,3 +36,29 @@ console.log(fruits.get('Apples')); // Output: 5
...
+
+- Samantha will `get` the next purchase order that she needs to fill from the map using the [`.get()` method](https://www.codecademy.com/resources/docs/javascript/map/get).
```
Comment (caupolicandiaz) [inline content/javascript/concepts/map/terms/get/get.md:45]:
```suggestion
- Samantha will `get` the next purchase order that she needs to fill from the map using the `.get()` method.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@dghalbr I've completed my initial review. Please update the entries, and tag me when you've finished making changes. Thanks

File: content/javascript/concepts/map/terms/delete/delete.md (hunk 3 above)
Comment (dghalbr) [reply content/javascript/concepts/map/terms/delete/delete.md:43]: I didn't even consider that! Great catch.

**Comment by dghalbr** [comment]
@caupolicandiaz thanks for the suggestions. I feel like I learned quite a bit! It's not often I create doc content like this. I consider the comments extremely valuable. I committed all of your suggestions as they were.

**Review by caupolicandiaz — APPROVED** [review]
@dghalbr Looks good

**Comment by SSwiniarski** [comment]
@dghalbr Congrats on your new edit. 

Live links here:
- https://www.codecademy.com/resources/docs/javascript/map/delete
- https://www.codecademy.com/resources/docs/javascript/map/get

**Pull request merged**